sources/packages/backend/apps/api/src/route-controllers/assessment/_tests_/e2e/assessment.institutions.controller.getAssessmentNOA.e2e-spec.ts

```diff
@@ -65,21 +67,147 @@ describe("AssessmentInstitutionsController(e2e)-getAssessmentNOA", () => {
       },
     );
     await db.msfaaNumber.save(currentMSFAA);
-    const application = await saveFakeApplicationDisbursements(db.dataSource, {
-      institution: collegeF,
-      institutionLocation: collegeFLocation,
-      student,
-      msfaaNumber: currentMSFAA,
-    });
+    const application = await saveFakeApplicationDisbursements(
+      db.dataSource,
+      {
+        institution: collegeF,
+        institutionLocation: collegeFLocation,
+        student,
+        msfaaNumber: currentMSFAA,
+      },
+      {
+        offeringIntensity: OfferingIntensity.fullTime,
+        createSecondDisbursement: true,
+      },
+    );
     const assessment = application.currentAssessment;
-    const [firstDisbursementSchedule] =
-      application.currentAssessment.disbursementSchedules;
+    const [firstDisbursementSchedule, secondDisbursementSchedule] =
+      assessment.disbursementSchedules;
 
-    const awards = {};
-    firstDisbursementSchedule.disbursementValues.forEach((disbursement) => {
-      awards[`disbursement1${disbursement.valueCode.toLowerCase()}`] =
-        disbursement.valueAmount;
-    });
+    const firstDisbursementScheduleAwards = createDisbursementScheduleAwards(
```

Comment:
Regarding the method `createDisbursementScheduleAwards`, IMO, it can also fall into the "First" point mentioned on https://github.com/bcgov/SIMS/pull/2365/files#r1346571749.
In general, for E2E test assertions, if possible to control the data, I would recommend to have more hard-coded `expect` verifications.